components/pd_client/src/lib.rs

```diff
@@ -149,9 +149,8 @@ pub trait PdClient: Send + Sync {
     }
 
     /// Gets Region info which the key belongs to.
-    fn get_region_info(&self, key: &[u8]) -> Result<RegionInfo> {
-        self.get_region(key)
-            .map(|region| RegionInfo::new(region, None))
+    fn get_region_info(&self, _key: &[u8]) -> Result<RegionInfo> {
```

Comment:
Will the implement of pd_client be affected by the change?

Reply:
No. It's implemented by all pd_clients except the dummy pd_client.